.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Run rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+          bundler-cache: true
+
+      - name: Run rubocop
+        run: bundle exec rubocop
```

Comment:
Once the RuboCop-provided Rake integration is available, you should call that rather than just `bundle exec rubocop`.

Reply:
Hi @jrgarcia Is there any customization needed like :
```
RuboCop::RakeTask.new(:rubocop) do |task|
  task.patterns = ['lib/**/*.rb']
  # only show the files with failures
  task.formatters = ['files']
  # don't abort rake on failure
  task.fail_on_error = false
end
```
OR  `RuboCop::RakeTask.new` is fine?

Reply:
I think the defaults are fine. You can just use `RuboCop::RakeTask.new`.

Reply:
done, I have updated the PR